src/ExporterInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sonata\Exporter;
+
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+interface ExporterInterface
+{
+    public function getResponse(string $format, string $filename, \Iterator $source): StreamedResponse;
```

Comment:
Here: https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Bridge/Exporter/AdminExporter.php#L24-L54

we're also using `getAvailableFormats`, so I think we should add it to the Interface too.

Reply:
Fixed.  Ok, I see, I will update AdminExporter too.